server/controllers/RecipeController.js

```diff
@@ -0,0 +1,96 @@
+import { Recipes as RecipeClass } from '../helpers/in-memory';
```

Comment:
This is most likely to be a model. Why not name it Recipe or RecipeModel instead?

Usually, you name models in their singular form as opposed to plural

Reply:
fixed